internal/version/version.go

```diff
@@ -51,6 +59,7 @@ type SemVer struct {
 }
 
 // AllowedVersions provides the Minimum SemVer per Service
+// TODO: I don't think this name is representative of what this struct now holds.
```

Comment:
maybe could be MinimumVersions? 🤷🏻‍♀️

Reply:
This struct stores versions for each process, not to be confused with the "processes" which it also holds, which contains "minimum" and "suggested" versions. Now those "processes" will additionally each hold a "rollout" which may or may not contain a version number depending on the expected relationship between the rollout "target version" (according to the design doc) and the other versions. It seems like there's probably some redundancy here that we can eliminate but I'm not familiar enough with the expected behavior for all these things to make that call.

Reply:
We can't use the MinimumVersions. It makes sense to bundle it later down the road.
Lets make an example. We have v0.20.x in production and want to update to v0.21.x. The correct config would be: Increase MinimumVersion from v0.19.0 to v0.20.0 and the suggested version from v0.20.0 to v0.21.0. For the next 2 weeks storage nodes can run both versions in parallel.

Reply:
I'm punting on doing any renaming in this PR. I'm just adding todos instead.